tornettools/generate_tgen.py

```diff
@@ -328,12 +371,24 @@ def __get_perf_clients(args, n_users):
     perf_clients = []
 
     n_perf = round(n_users * args.torperf_scale)
-    for i in range(n_perf):
+
+    # split the perf clients between onion-service clients and non-onion-service (public) clients
+    n_hs_perf = round(n_perf * args.onion_service_traffic_scale)
```

Comment:
Should we use `ceil` instead of `round` to make sure we always have at least one as long as `args.onion_service_traffic_scale` is positive?

Reply:
Added.